apps/emqx_management/src/emqx_mgmt_api_acl.erl

```diff
@@ -0,0 +1,47 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2021 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+-module(emqx_mgmt_api_acl).
+
+-include("emqx_mgmt.hrl").
+
+-rest_api(#{name   => clean_acl_cache_all,
+            method => 'DELETE',
+            path   => "/acl-cache",
+            func   => clean_all,
+            descr  => "Clean acl cache on all nodes"}).
+
+-rest_api(#{name   => clean_acl_cache_node,
+            method => 'DELETE',
+            path   => "/:atom:node/acl-cache",
+            func   => clean_node,
+            descr  => "Clean acl cache on specific node"}).
+
+-export([ clean_all/2
+        , clean_node/2
+        ]).
+
+clean_all(_Bindings, _Params) ->
+    case emqx_mgmt:clean_acl_cache_all() of
+      ok -> minirest:return();
+      {error, Reason} -> minirest:return({error, ?ERROR1, Reason})
```

Comment:
sorry, approved too soon.
```
./emqx_mgmt_api_acl.erl
  40: The call minirest:return({'error',101,[{_,_},...]}) will never return since it differs in the 1st argument from the success typing arguments: ('ok' | {'error',atom() | binary() | tuple()} | {'ok',_} | {'error',_,atom() | binary() | tuple()} | {'ok',_,_})
```

Reply:
`minirest:return({error, Code, Reason})` is a HTTP 500 error.

Reply:
{IntegerHTTPCode, <<"text....">>} will return the integer code to the http client.

Reply:
I think here we should raise a 500, so just `minirest:return({error, Reason})` is good.

Reply:
will check/fix all of these. still working on figuring out the functional tests in here ;)

Reply:
I had a closer look at the codes.
We may need to make good use of it.
so please format `Reason` to `binary()` to make `minirest:return/1` happy.
i.e. `iolist_to_binary(io_lib:format("~0p", [Reason]))`

Reply:
what if we fix it on the minirest level so it can take list as Reason in `return/1` ex https://github.com/sh3d2/minirest/pull/1/files

Reply:
fixed in minirest and bumped to 0.3.5